helper/Makefile.am

```diff
@@ -15,6 +15,7 @@ helperinclude_HEADERS = \
 		  include/odp/helper/chksum.h\
 		  include/odp/helper/odph_debug.h \
 		  include/odp/helper/eth.h\
+		  include/odp/helper/gtpu.h\
```

Comment:
I suppose this header file should be called just gtp.h.

Reply:
Ack